### PR TITLE
Gh actions selenium

### PR DESCRIPTION
**Pull request by TomWoodward**
for: openstax/unified#1193

### CODE REVIEW

File: .github/workflows/ci-selenium.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    runs-on: ubuntu-latest
+    container: openstax/rex-web:CI
+    steps:
+    - uses: actions/checkout@v2
+    - name: find deployment
+      env:
+        GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        . concourse/common/get_review_environment
+        review_url=$(get_review_environment)
```
Comment (PiotrKozlowski) [inline .github/workflows/ci-selenium.yml:16]: I'm not really familiar with bash. Is this `${get_review_environment}` a returned valud from `. concourse/common/get_revview_environment`?

**Review by PiotrKozlowski — COMMENTED** [review]
In logs: https://github.com/openstax/rex-web/pull/721/checks?check_run_id=847303711
I can see:
`jq: error (at <stdin>:4): Cannot index object with number`
I don't know however on which `jq` call this is throwing an error, but I can see that in the line 29 in `get_review_environment` file `.[0].state` is not in `''` like it is in the other statements so my guess would be to check if wrapping it like `jq -r '.[0].state'` will help and if not then add some console logs to figure out why this is throwing an error

File: .github/workflows/ci-selenium.yml (hunk 1 above)
Comment (TomWoodward) [reply .github/workflows/ci-selenium.yml:16]: `get_review_environment` is a function defined in `concourse/common/get_review_environment`. the `. <filename>` sources the file so you can call the function in it.

the `jq: error (at <stdin>:4): Cannot index object with number` error happens when the github api is rate limited. i'm going to do some testing today about that

**Comment by TomWoodward** [comment]
@PiotrKozlowski this is ready for review now, sorry i didn't make it as a draft to begin with

File: concourse/common/get_review_environment [hunk 2]
```diff
@@ -7,33 +7,42 @@ then
 fi
 
 function github {
-  curl -s -H "Authentication: token $GITHUB_ACCESS_TOKEN" "https://api.github.com/$1"
+  response=$(curl -s -H "Authentication: token $GITHUB_ACCESS_TOKEN" "https://api.github.com/$1")
+  echo "https://api.github.com/$1">&2
+  echo "$response">&2
+  echo "$response"
```
Comment (PiotrKozlowski) [inline concourse/common/get_review_environment:13]: do we want to leave all of these 'echo`es in this file?

**Review by PiotrKozlowski — APPROVED** [review]

File: concourse/common/get_review_environment (hunk 2 above)
Comment (TomWoodward) [reply concourse/common/get_review_environment:13]: yea they are captured in the build logs and have been helpful for debugging

**Pull request merged**